cool.py

```diff
@@ -551,8 +583,20 @@ def locate_results(results, col, row):
         return results_df
 
     #: calculate centroid in pixels
-    results_df["x_centroid_px"] = (results_df["xmin"] + results_df["xmax"]) / 2
-    results_df["y_centroid_px"] = (results_df["ymin"] + results_df["ymax"]) / 2
+    results_df["centroid_x_px"] = (results_df["xmin"] + results_df["xmax"]) / 2
+    results_df["centroid_y_px"] = (results_df["ymin"] + results_df["ymax"]) / 2
+
+    results_df.rename(
```

Comment:
curious: what are the naming constraints for pgsql that you have to rename columns to operate within?

Reply:
so, xmax was throwing an error that it was a system column or something? Then class and name are reserved words and required quotes so I made everything a bit more conventional.